samples/otlptraceexport/tsconfig.json

```diff
@@ -1,8 +1,12 @@
 {
-  "extends": "../../node_modules/gts/tsconfig-google.json",
+  "extends": "gts/tsconfig-google.json",
   "compilerOptions": {
     "rootDir": ".",
     "outDir": "build",
     "esModuleInterop": true
-  }
+  },
```

Comment:
Without this, typescript tries to compile the `build/` directory and complains about overwriting input files